distribution/src/bin/elasticsearch-cli

```diff
@@ -7,7 +7,7 @@ source "`dirname "$0"`"/elasticsearch-env
 IFS=';' read -r -a additional_sources <<< "$ES_ADDITIONAL_SOURCES"
 for additional_source in "${additional_sources[@]}"
 do
-  source "`dirname "$0"`"/$additional_source
+  source $ES_HOME/bin/$additional_source
```

Comment:
We probably need quotes here since ES_HOME could have spaces. Can you verify this, and quote if needed.

Also, does Windows have the same problem and need a fix too?

Reply:
> Also, does Windows have the same problem and need a fix too?

we use: 
```
if defined ES_ADDITIONAL_SOURCES (
  for %%a in ("%ES_ADDITIONAL_SOURCES:;=","%") do (
    call "%~dp0%%a"
  )
)
```

`%~dp0` will _always_ expand to the full path where the batch file is located and not one relative to where the batch script is executed from, so it will not be affected by the cd that happens when we source `elasticsearch-env.bat` . That said, I'd have nothing against a manual confirmation, I don't have access to a windows machine right now :/

Reply:
That looks right to me, thanks for checking.